packages/ui/react-ui-grid/src/Grid/Grid.stories.tsx

```diff
@@ -4,41 +4,46 @@
 
 import '@dxos-theme';
 
-import { type StoryObj } from '@storybook/react';
+import { type Meta, type StoryObj } from '@storybook/react';
 import React, { type MouseEvent, type MutableRefObject, useCallback, useRef, useState } from 'react';
 
 import { defaultSizeRow } from '@dxos/lit-grid';
 import { faker } from '@dxos/random';
 import { DropdownMenu } from '@dxos/react-ui';
-import { PopoverCombobox } from '@dxos/react-ui-searchlist';
+import { PopoverCombobox, type PopoverComboboxRootProps } from '@dxos/react-ui-searchlist';
 import { withTheme } from '@dxos/storybook-utils';
 
-import { Grid, type GridContentProps, type GridRootProps } from './Grid';
+import { Grid, type GridEditing, type GridContentProps, type GridRootProps } from './Grid';
 
-export { defaultSizeCol, defaultSizeRow } from '@dxos/lit-grid';
+const storybookItems = faker.helpers.uniqueArray(faker.commerce.productName, 16);
 
-type StoryGridProps = GridContentProps & Pick<GridRootProps, 'onEditingChange'>;
+type GridStoryProps = GridContentProps & Pick<GridRootProps, 'onEditingChange'>;
 
-const storybookItems = faker.helpers.uniqueArray(faker.commerce.productName, 16);
+const GridStory = ({ initialCells, ...props }: GridStoryProps) => {
+  const triggerRef = useRef<HTMLButtonElement>(null) as MutableRefObject<HTMLButtonElement>;
 
-const storybookInitialCells = (value?: string) => ({
-  grid: {
-    '1,1': {
-      accessoryHtml:
-        '<button class="dx-button is-6 pli-0.5 min-bs-0 absolute inset-block-1 inline-end-1" data-story-action="menu"><svg><use href="/icons.svg#ph--arrow-right--regular"/></svg></button>',
-      value: 'Weekly sales report',
-    },
-    '2,2': {
-      accessoryHtml: `<dx-grid-multiselect-cell ${value ? `values='${JSON.stringify([{ label: value }])}'` : ''} placeholder="Select…"></dx-grid-multiselect-cell>`,
-    },
-  },
-});
+  const [cells, setCells] = useState<GridContentProps['initialCells']>(initialCells);
 
-const StoryGrid = ({ onEditingChange, ...props }: StoryGridProps) => {
-  const [menuOpen, setMenuOpen] = useState(false);
+  const [editing, setEditing] = useState<GridEditing>(null);
+  const handleEditingChange = useCallback<NonNullable<GridRootProps['onEditingChange']>>((event) => {
+    // TODO(burdon): Not working?
+    setEditing(event ? { index: event.index, initialContent: '' } : null);
+  }, []);
+
+  // Multiselect
   const [popoverOpen, setPopoverOpen] = useState(false);
   const [multiSelectValue, setInternalMultiselectValue] = useState('');
-  const triggerRef = useRef<HTMLButtonElement>(null) as MutableRefObject<HTMLButtonElement>;
+  const setMultiselectValue = useCallback<NonNullable<PopoverComboboxRootProps['onValueChange']>>((nextValue) => {
+    setInternalMultiselectValue(nextValue);
+    setCells((cells) => {
+      // TODO(burdon): How can we get the cell address to update?
```

Comment:
`closestCell` exported by `lit-grid` is designed to give you the cell index/position/address given an event target. The cell is needed for the popover, so the event target should be available within a related closure.

Reply:
See e.g. `GridSheet` for how that’s used.

Reply:
Can you update this branch so that the cell is editable.